tests/unit/App.spec.ts

```diff
@@ -3143,7 +3143,7 @@ describe('App as a completed continuation in', () => {
     // verify loaded filing
     expect(filingHistoryListStore.filings.length).toBe(1)
     expect(filingHistoryListStore.filings[0].businessIdentifier).toBe('T123456789')
-    expect(filingHistoryListStore.filings[0].displayName).toBe('BC Limited Company Continuation Application')
+    expect(filingHistoryListStore.filings[0].displayName).toBe('BC Limited Company Continuation Authorization')
     expect(filingHistoryListStore.filings[0].name).toBe('continuationIn')
     expect(filingHistoryListStore.filings[0].status).toBe('COMPLETED')
```

Comment:
No. A completed continuation filing should be labelled an application. (It should display in the Filings History list.)

Reply:
This will be answered by my answer to your other comment, below.

Reply:
Resolving for now. Please come back to this if needed.